buildSrc/src/main/kotlin/io/spine/internal/dependency/Spine.kt

```diff
@@ -85,7 +85,7 @@ class Spine(p: ExtensionAware) {
         /**
          * The version of `mc-java` to use.
          */
-        const val mcJava = "2.0.0-SNAPSHOT.106"
+        const val mcJava = "2.0.0-SNAPSHOT.105"
```

Comment:
This is weird. Please review, why we downgrade here.

Reply:
The latest `config` declares `105`. For some reasons, this repo declared a newer version of `mc-java`, which in any case is not used here. So, it's safe to left it match to the latest `config`.

Reply:
@alexander-yevsyukov have you set this "newer" McJava version here on purpose?

@yevhenii-nadtochii we need to answer this question now to avoid any undesired side effects. Next time when I ask, please DO investigate.

Reply:
I don't remember something special. We need McJava in this repository in order to generate from from proto types with all our features. I'll set it back, once the journey around #34 finishes.